packages/hooks/src/useGreeting/index.en-US.md

```diff
@@ -0,0 +1,34 @@
+---
+nav:
+path: /hooks
+---
+
+# useGreeting
+
+A hook that returns a greeting message based on the current time.
+
+## Examples
+
+### Default usage
+
+<code src="./demo/demo1.tsx" />
+
+## API
+
+```typescript
+const greeting = useGreeting();
+```
+
+### Params
+
+| Property  | Description                                                                                          | Type      | Default   |
+| --------- | ---------------------------------------------------------------------------------------------------- | --------- | --------- |
+| prefix    | add optionally a string before the state of the day                                                  | `string`  |           |
+| suffix    | add optionally a string after the state of the day                                                   | `string`  |           |
+| transform | Optionally transform (morning/afternoon/evening/night) into uppercase/capitalize first letter letter | `boolean` | undefined |
+
```

Comment:
Remember to update here synchronously, as in the `.zh-CN.md`

Reply:
resolved